packages/cyberstorm/package.json

```diff
@@ -38,6 +38,7 @@
     "react": "^18.2.0",
     "react-data-table-component": "^7.5.3",
     "react-dom": "^18.2.0",
+    "react-hook-form": "^7.48.2",
```

Comment:
Not really related to this line, but: have we considered whether the forms should be defined in the Cyberstorm package or in the nextjs app? IDK right now what the ramifications of the decision would be (if any). Based on a gut feeling I'd say the forms are unlikely to be very reusable and relate more to the app than the components.

Reply:
Have not really talked about this yet; I had an idea that we should move the layouts to the nextjs app. So then these forms would move there also. I don't want to do anything towards that yet though, not at least before we've come to an unified conclusion on the matter.

Let's think about this after the MVP 👍